### PR TITLE
Config Split

### LINKED ISSUE #49: Separate config for slack and webserver
Separate out config where only needed in 1 part of the app.

### PR DESCRIPTION
**Pull request by samcorky**
Split config

### CODE REVIEW

**Comment by samcorky** [comment]
blocked by #68 

I should've based this branch off of that one

**Comment by sjefferson99** [comment]
@sam57719 Merge branch 68 into this one and keep it open. We can merge 68 into develop and then 69 on top later, or just merge 69 in and discard 68 as we need.

**Comment by samcorky** [comment]
@sjefferson99  this now should be able to go in separatley to the docker branches #68 - found better way of sorting the injectable issue

File: docker-compose.yml
Comment (sjefferson99) [inline docker-compose.yml:1]: @sam57719 Does this mean that env var export is no longer supported and tokens must now be provided with .env?

Comment (samcorky) [reply docker-compose.yml:1]: @sjefferson99 

ENV var will still be supported as it currently is - for any values that are set in the docker compose like this
```
    environment:
      ...

      # Passed in from HOST
      - SLACK_APP_TOKEN
      - SLACK_BOT_TOKEN
```

Comment (samcorky) [reply docker-compose.yml:1]: .env will be cleaner - will update readme :)

Comment (sjefferson99) [reply docker-compose.yml:1]: Confirmed this is just removing from the webserver compose element, that no longer needs it.

**Review by sjefferson99 — APPROVED** [review]
All fine.

**Pull request merged**